docs/configuration.rst

```diff
@@ -104,6 +104,25 @@ ACCOUNT_USERNAME_REQUIRED (=True)
   `ACCOUNT_AUTHENTICATION_METHOD` is set to `email`. Set to `False`
   when you do not wish to prompt the user to enter a username.
 
+ACCOUNT_LOGIN_CALLBACK_PROXY (="")
+  Redirect the login callback to this remote server or endpoint, instead of the
+  default endpoint on the local server. The remote server should be a second
+  instance of the local server, at a dedicated URL. By enabling
```

Comment:
Yes, no, and yes.
Yes: The url bindings need to be consistent, and the same OAuth2 providers need to be enabled. Using the same instance of this server guarantees that.
No: One could code the bindings manually.
Yes: But who wants that!? It's so much easier to just launch the same server twice.